scripts/docker/nginx/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM nginx:1-alpine
+FROM nginx:alpine
```

Comment:
@afabiani I would use a fixed version here instead of the latest, [as we do with the GeoNode Project](https://github.com/GeoNode/geonode-project/blob/master/docker/nginx/Dockerfile#L1)